android/app/src/main/java/app/organicmaps/bookmarks/BookmarkCategoriesFragment.java

```diff
@@ -254,7 +256,19 @@ public void onImportButtonClick()
 
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
       intent.putExtra(DocumentsContract.EXTRA_EXCLUDE_SELF, true);
-    startActivityForResult(intent, REQ_CODE_IMPORT_DIRECTORY);
+    PackageManager packageManager = requireActivity().getPackageManager();
+    if (intent.resolveActivity(packageManager) != null) {
+      startActivityForResult(intent, REQ_CODE_IMPORT_DIRECTORY);
+    } else {
+      showNoFileManagerError();
+    }
```

Comment:
1. One-liners don't need braces.
2. If braces are used, they should be on a new line (check other code style and documentation/related style config files in the repo).
```suggestion
    if (intent.resolveActivity(packageManager) != null)
      startActivityForResult(intent, REQ_CODE_IMPORT_DIRECTORY);
    else
      showNoFileManagerError();
```